Website/src/rendering/src/assets/css/components/InformationPageHero.css

```diff
@@ -11,13 +11,17 @@
     background: linear-gradient(90deg, rgba(255, 255, 255, 1) 50%, rgba(35, 35, 35, 1) 50%);
   }
 
+  .background-container {
```

Comment:
One of the changes in this file broke the SpeakerInformationPageHero image fit:
![image](https://user-images.githubusercontent.com/2000200/143252061-5797a367-c484-44f5-a60b-3c434e95e52f.png)